capstone/src/main/resources/static/event.js

```diff
@@ -46,6 +154,49 @@ function loadEventInfo() {
       document.getElementById("event-id").value = data.datastoreID;
     });
   }
+}
+
+/**
+ * Create a page to view event details
+ * @param eventId event's datastore id
+ */
+function fillDetails(event) {
+  const eventName = document.getElementById("eventName");
+  eventName.innerText = event.eventTitle;
 
+  var date = new Date(event.eventDateTime);
+  const eventTime = document.getElementById("eventTime");
+  eventTime.innerText = date.toString().substring(0, 21);
+
+  const eventLocation = document.getElementById("eventLocation");
+  eventLocation.innerText = event.eventLatitude;
+
+  const eventDescription = document.getElementById("eventDescription");
```

Comment:
Similarly, there could be a helper function `setElementText()` taking the element ID and new contents.